codegen/JsonUtils/src/test/java/JsonParserTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+
+public class JsonParserTest {
+    final String Q6_JSON = "C:\\Users\\Danish Ibrahim\\Desktop\\gui-codegen\\codegen\\JsonUtils\\src\\test\\resources\\Q6.json";
```

Comment:
The test should not be an absolute path, or you should have a script that automatically matching a path in a different machine.

Reply:
done

Reply:
I think the issue is the user needs to manually change the path in order to perform the test, should have a script that allows the user to set the part as the project folder path.  If more tests are added later, then all of them need to be changed after pulling the project from GitHub.

Reply:
No, later it will look like this, I am just waiting for more jsons to be ready:
`    @Test
    void parseQ6() throws Exception {
        JsonParser.parse(Q6_JSON);
    }
@Test
    void parseQ1() throws Exception {
        JsonParser.parse(Q1_JSON);
    }
`etc

Reply:
this way we can run each test separately and know which one fails, if any do